sdk/src/dataMapper/UtilsDataMapper.ts

```diff
@@ -16,27 +14,15 @@ export default class UtilsDataMapper extends BaseDataMapper<object, unknown, unk
   }`;
 
   async getModulesNumber() {
-    return this.web3Client.readContract({
-      abi: abiModuleRegistry,
-      address: this.conf.moduleRegistryAddress,
-      functionName: "getModulesNumber",
-    });
+    return await this.getCounterForType("module");
   }
 
   async getPortalsCount() {
-    return this.web3Client.readContract({
-      abi: abiPortalRegistry,
-      address: this.conf.portalRegistryAddress,
-      functionName: "getPortalsCount",
-    });
+    return await this.getCounterForType("portal");
   }
 
   async getSchemasNumber() {
-    return this.web3Client.readContract({
-      abi: abiSchemaRegistry,
-      address: this.conf.schemaRegistryAddress,
-      functionName: "getSchemasNumber",
-    });
+    return await this.getCounterForType("schema");
   }
```

Comment:
Actually, this looks like a duplicate of the functions we already have in the `DataMapper`, so let's remove them from here.